infra/airflow/Dockerfile

```diff
@@ -12,6 +12,8 @@ COPY requirements.txt /opt/airflow
 
 # working in the /opt/airflow
 WORKDIR /opt/airflow
+RUN wget -c https://github.com/RMLio/rmlmapper-java/releases/download/v5.0.0/rmlmapper-5.0.0-r362-all.jar -P .rmlmapper/
+RUN mv .rmlmapper/rmlmapper-5.0.0-r362-all.jar .rmlmapper/rmlmapper.jar 2>/dev/null
```

Comment:
not sure we shall crowd `/opt/airflow` with the RMLMapper
a cleaner alternative would be to place it into `/opt/rmlmapper` but this requires setting the right env variables